src/components/Markdown.svelte

```diff
@@ -13,6 +13,15 @@
         start ? `start="${start}"` : ""
       } class="mzp-u-list-styled">${body}</${outerEl}>`;
     },
+    heading(text, level, raw) {
+      const id = raw
+        .toLowerCase()
+        .trim()
+        .replace(/<[!\/a-z].*?>/gi, '')
```

Comment:
do we need the `\/` escape?

```suggestion
        .replace(/<[!a-z].*?>/gi, '')
```

Reply:
Should have noted that:
I took this code from the recommended marked addon here: https://github.com/markedjs/marked-gfm-heading-id/blob/main/src/index.js (no need for a new dep for such simple functionality).

This removes everything that looks roughly like an HTML tag, that's why `/` is matched there as well.